registration/templates/registration/activation_complete_admin_pending.html

```diff
@@ -0,0 +1,22 @@
+{% extends "registration/registration_base.html" %}
```

Comment:
Ah, I thought you wanted screenshots of the templates used create the new emails (towards admins, user)

Ok, the first new template is `activation_complete_admin_pending.html`
![screenshot from 2016-06-21 10 18 19](https://cloud.githubusercontent.com/assets/14074326/16221134/c51c54fc-3799-11e6-8004-b7a1a8f27e29.png)

And the second is 
![screenshot from 2016-06-21 10 19 18](https://cloud.githubusercontent.com/assets/14074326/16221147/da742ac8-3799-11e6-850e-73022522eca2.png)